stl/inc/format

```diff
@@ -2472,7 +2400,7 @@ inline constexpr bool _Is_two_tuple<tuple<_Ty, _Uty>> = true;
 
 template <_RANGES input_range _Rng>
     requires same_as<_Rng, remove_cvref_t<_Rng>>
-constexpr range_format format_kind<_Rng> = []() consteval {
+inline constexpr range_format format_kind<_Rng> = []() consteval {

```

Comment:
again, this seems to be a redundant inline specifier

Reply:
Ditto "sadly not redundant for MSVC".